tests/hijack/async.js

```diff
@@ -81,6 +84,9 @@ Tinytest.add(
       return acc;
     }, {});
 
+    console.log('compute', events.compute);
+    console.log('async', events.async);
```

Comment:
Are these console.log's needed?

Reply:
Yes, if the test fails again we will understand why